packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Neo4jPrunsrv.ps1

```diff
@@ -103,6 +103,7 @@ Function Get-Neo4jPrunsrv
           '--StartMethod=start',
           "`"--StartPath=$($Neo4jServer.Home)`"",
           "`"--StartParams=--config-dir=$($Neo4jServer.ConfDir)`"",
+          "`"++StartParams=--home-dir=$($Neo4jServer.Home)`"",
```

Comment:
@srbaker I don't remember -- do you need to change `--` to `++` on the other `StartParams` as well?